CHANGELOG.md

```diff
@@ -10,6 +10,11 @@
 - `[pretty-format]` Support React.Suspense ([#8180](https://github.com/facebook/jest/pull/8180))
 - `[jest-snapshot]` Indent inline snapshots ([#8198](https://github.com/facebook/jest/pull/8198))
 - `[jest-config]` Support colors in `displayName` configuration ([#8025](https://github.com/facebook/jest/pull/8025))
+- `[@jest/core]` Move `testSequencer` to individual package `@jest/test-sequencer` ([#8223](https://github.com/facebook/jest/pull/8223))
```

Comment:
@SimenB something like `@jest/core/@jest/test-sequencer` like we've done previously for changelog entries that affect multiple packages doesn't play well with scoped packages, should we comma separate them?

Reply:
(Multiple lines definitely seems like something we should avoid)

Reply:
Agreed. What about `[@jest/core, jest-config]` etc? In this case I think 2 lines are fine - 1 for extracting the sequencer to a separate package, and one for making it customizable through config

Reply:
Yes that's the format I thought of :+1: 
And two lines with the two distinct sentences are of course fine